Cristiandi/basic-acl-api
# Feature/35

## What did you do?:

resolve #35
resolve #37
resolve #38
resolve #40 
resolve #41

## How can QA verify the change?:
Go to the root of the project and tun `npm run start:dev` to see it.

***Is the change ready to review?:*** yep
